src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -54,7 +56,12 @@ public class Drive extends SubsystemBase {
   private final Module[] modules = new Module[4]; // FL, FR, BL, BR
 
   private SwerveDriveKinematics kinematics = new SwerveDriveKinematics(getModuleTranslations());
+
+  @Getter
+  @Setter
```

Comment:
This blows my mind that it exists lol

Reply:
Not sure if I like it or not yet but think I'll get used to it.